lib/setupmanagers/qemuhck/devices/iommu.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "iommu",
   "command_line": [
-    "-device intel-iommu,intremap=on,aw-bits=48,caching-mode=on,device-iotlb=on"
+    "-device intel-iommu,intremap=on,aw-bits=48,caching-mode=off,eim=on"
```

Comment:
For `device-iotlb`, QEMU documentation says:
> This enables device-iotlb capability for the emulated VT-d device.  So far virtio/vhost should be the only real user for this parameter, paired with ats=on configured for the device.

`iommu_device_param` still has `ats=on`, which probably does not make sense.

Reply:
Added device-iotlb.